dojo/tools/twistlock/parser.py

```diff
@@ -132,7 +112,7 @@ def get_item(vulnerability, test):
         # If we're dealing with a license finding, there will be no cvssScore
         if vulnerability['severity'] == 'important':
             severity = "High"
-        elif vulnerability['severity'] == 'moderate':
+        elif vulnerability['severity'].lower() == 'moderate':
             severity = "Medium"
         else:
             severity = vulnerability['severity'].title()
```

Comment:
not sure what values title can have but the parser needs to make sure only to insert severities known to dojo, otherwise there can be unpredictable results.

Reply:
`.title()` do an uppercase for the first letter and a lower case for the rest of the string. It is a good function for severity in parsers code as for now, we have `Info`, `High`, etc... as values.
My advice here is just to encapsulate the logic of converting severity values from the format to Dojo values in a separate function. And just call it when creating the finding, like this:
```Python
severity = self.convert_severity(vulnerability['severity'])
```

Reply:
Created a [function](https://github.com/DefectDojo/django-DefectDojo/pull/3532/files#diff-3176294e4014378ed7fcff6167cc0a0e5e077c2dd816ebed034e2c5c23e7e754R148) to properly parse the severity. Modified both CSV and JSON parser to use the function.